.gitignore

```diff
@@ -16,7 +16,6 @@ Thumbs.db
 
 # Wordpress
 wp-content/*
-!wp-content/themes/
```

Comment:
Just pushed a commit that adds an example `.gitignore` as well as a direct download one-liner to get the example directory only.